controller/src/main/java/ru/mail/dimaushenko/controller/filter/LoginFilter.java

```diff
@@ -0,0 +1,49 @@
+package ru.mail.dimaushenko.controller.filter;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import static ru.mail.dimaushenko.constants.AttributeConstants.ATTRIBUTE_NAME_ERRORS;
+import static ru.mail.dimaushenko.constants.ErrorsConstants.ERROR_KEY_USERNAME_NOT_FOUND;
+import static ru.mail.dimaushenko.constants.ErrorsConstants.ERROR_VALUE_USERNAME_NOT_FOUND;
+import ru.mail.dimaushenko.service.UserService;
+import ru.mail.dimaushenko.service.impl.UserServiceImpl;
+import static ru.mail.dimaushenko.constants.ParameterConstants.PARAMETER_USERNAME;
+
+public class LoginFilter implements Filter {
+
+    private final String METHOD_GET = "GET";
+    private final String METHOD_POST = "POST";
```

Comment:
better to use constant https://docs.oracle.com/javaee/6/api/javax/ws/rs/HttpMethod.html